lib/cocoapods/target.rb

```diff
@@ -305,6 +305,18 @@ def mark_application_extension_api_only
       @application_extension_api_only = true
     end
 
+    # @return [Pathname] The absolute path of the embed frameworks script.
+    #
+    def embed_frameworks_script_path
```

Comment:
why does this need to be here?

Reply:
this was moved from `AggregateTarget`, but I think I can move it back as I ended up reworking what I was going to use this for.